### PR TITLE
Update vets-website to v0.1.1173 & update composer nodejs from 10 to 14.

### PR DESCRIPTION
**Pull request by indytechcook**
## Description

Update vets-website to lastest tag.

---
This change is [Reviewable](https://reviewable.io/reviews/department-of-veterans-affairs/va.gov-cms/3600)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -140,7 +140,7 @@
         "symfony/config": "^3.4",
         "symfony/finder": "^3",
         "symfony/phpunit-bridge": "^5.1",
-        "va-gov/web": "^0.1",
+        "va-gov/web": "^0.1.1173",
```
Comment (ElijahLynn) [inline composer.json:143]: Are we intentionally hardcoding the version here in composer.json or was this unintended?

**Comment by ElijahLynn** [comment]
I was expecting a failure on composer install for this, but that didn't happen, so looking at that now.

**Comment by indytechcook** [comment]
> I was expecting a failure on composer install for this, but that didn't happen, so looking at that now.

You already fixed this on the devshop server: https://github.com/department-of-veterans-affairs/va.gov-cms/pull/3530#issuecomment-730759981

File: composer.json (hunk 1 above)
Comment (indytechcook) [reply composer.json:143]: My bad.  I ran `lando composer require va-gov/web` and it added this.

**Comment by indytechcook** [comment]
I set this to pull the specific commit https://github.com/department-of-veterans-affairs/vets-website/commit/4749648bd8506c6e2432f27d395fa5cfd75c19a11. Tomorrow I will update it back to `^0.1`

**Review by ElijahLynn — APPROVED** [review]

**Pull request merged**